.github/workflows/test.yaml

```diff
@@ -13,51 +13,11 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          submodules: true
-
-      # Install Catlin linting tool
-      - name: Install linting tools
-        run: |
-          set -Eeu
-          GOBIN=/usr/local/bin/ go install github.com/openshift-pipelines/tektoncd-catalog/cmd/catalog-cd@latest
-          git clone https://github.com/tektoncd/catlin
-          cd catlin
-          make bin/catlin
-          sudo cp bin/catlin /usr/local/bin
-          catlin --help
```

Comment:
Why are we removing the portion of linting using Catlin? @savitaashture

Reply:
good question!!

I did check for all the repos and i don't see anywhere we are doing linting as well i remember @vdemeester saying that something related to catlin tool 

So just removed from Openshift as well and thought to bring this point in the ecosystem discussion call to have a consistent linting across all the repos for yamls

Reply:
Yeah but I remember @vdemeester told that we require to run some form of linting on the results of the helm template and we used Catlin as it was upstream. This linting was added because there was a JIRA issue that required that our tasks (or the tasks in ecosystem catalog) should contain Red Hat supported images only and to ensure this linting was essential. @savitaashture

Reply:
But I think @vdemeester can provide a better context to this  than me@savitaashture

Reply:
@Senjuti256 

I removed only catalog-cd tool part as there is no lint and kept catalin lint as it is